src/analysis/normed_space/spectrum.lean

```diff
@@ -131,6 +140,125 @@ end
 
 end resolvent_deriv
 
+section one_sub_smul
+
+open continuous_multilinear_map ennreal formal_multilinear_series
+open_locale nnreal ennreal
+
+variables
+[nondiscrete_normed_field 𝕜] [normed_ring A] [normed_algebra 𝕜 A]
+
+variable (𝕜)
+/-- In a Banach algebra `A` over a nondiscrete normed field `𝕜`, for any `a : A` the
+power series with coefficients `a ^ n` represents the function `(1 - z • a)⁻¹` in a disk of
+radius `∥a∥₊⁻¹`. -/
+lemma inverse_one_sub_smul_has_fpower_series_on_ball [complete_space A] (a : A) :
+  has_fpower_series_on_ball (λ z : 𝕜, ring.inverse (1 - z • a))
+    (λ n, continuous_multilinear_map.mk_pi_field 𝕜 (fin n) (a ^ n)) 0 (∥a∥₊)⁻¹ :=
+{ r_le :=
+  begin
+    refine le_of_forall_nnreal_lt (λ r hr, le_radius_of_bound_nnreal _ (max 1 ∥(1 : A)∥₊) (λ n, _)),
+    rw [←norm_to_nnreal, norm_mk_pi_field, norm_to_nnreal],
+    cases n,
+    { simp only [le_refl, mul_one, or_true, le_max_iff, pow_zero] },
+    { refine le_trans (le_trans (mul_le_mul_right' (nnnorm_pow_le' a n.succ_pos) (r ^ n.succ)) _)
+        (le_max_left _ _),
+      { by_cases ∥a∥₊ = 0,
+        { simp only [h, zero_mul, zero_le', pow_succ], },
+        { rw [←coe_inv h, coe_lt_coe, nnreal.lt_inv_iff_mul_lt h] at hr,
+          simpa only [←mul_pow, mul_comm] using pow_le_one' hr.le n.succ } } }
+  end,
+  r_pos := ennreal.inv_pos.mpr coe_ne_top,
+  has_sum := λ y hy,
+  begin
+    have norm_lt : ∥y • a∥ < 1,
+    { by_cases h : ∥a∥₊ = 0,
+      { simp only [nnnorm_eq_zero.mp h, norm_zero, zero_lt_one, smul_zero] },
+      { have nnnorm_lt : ∥y∥₊ < ∥a∥₊⁻¹,
+          by simpa only [←coe_inv h, mem_ball_zero_iff, metric.emetric_ball_nnreal] using hy,
+        rwa [←coe_nnnorm, ←real.lt_to_nnreal_iff_coe_lt, real.to_nnreal_one, nnnorm_smul,
+          ←nnreal.lt_inv_iff_mul_lt h] } },
+    simpa [←smul_pow, (normed_ring.summable_geometric_of_norm_lt_1 _ norm_lt).has_sum_iff]
+      using (normed_ring.inverse_one_sub _ norm_lt).symm,
+  end }
+
+variable {𝕜}
+lemma is_unit_one_sub_smul_of_lt_inv_radius {a : A} {z : 𝕜} (h : ↑∥z∥₊ < (spectral_radius 𝕜 a)⁻¹) :
+  is_unit (1 - z • a) :=
+begin
+  by_cases hz : z = 0,
+  { simp only [hz, is_unit_one, sub_zero, zero_smul] },
+  { let u := units.mk0 z hz,
+    suffices hu : is_unit (u⁻¹ • 1 - a),
+    { rwa [is_unit.smul_sub_iff_sub_inv_smul, inv_inv u] at hu },
+    { rw [units.smul_def, ←algebra.algebra_map_eq_smul_one, ←mem_resolvent_set_iff],
+      refine mem_resolvent_set_of_spectral_radius_lt _,
+      rwa [units.coe_inv', normed_field.nnnorm_inv, coe_inv (nnnorm_ne_zero_iff.mpr
+        (units.coe_mk0 hz ▸ hz : (u : 𝕜) ≠ 0)), lt_inv_iff_lt_inv] } }
+end
+
+/-- In a Banach algebra `A` over `𝕜`, for `a : A` the function `λ z, (1 - z • a)⁻¹` is
+differentiable on any closed ball centered at zero of radius `r < (spectral_radius 𝕜 a)⁻¹`. -/
+theorem inverse_one_sub_smul_differentiable_on [complete_space A] {a : A} {r : ℝ≥0}
```

Comment:
```suggestion
theorem differentiable_on_inverse_one_sub_smul [complete_space A] {a : A} {r : ℝ≥0}
```
This seems to be the convention for differentiability statements (see e.g. `real.differentiable_on_log`)

Reply:
I changed the one for `has_fpower_series_on_ball` too.